sdk/compatibility/WORKSPACE

```diff
@@ -213,6 +213,55 @@ filegroup(
     nix_file = "@daml//nix:bazel.nix",
     nix_file_deps = common_nix_file_deps,
     repositories = dev_env_nix_repos,
+) if not is_windows else None
```

Comment:
It seems you can remove `postgresql_nix` completely as it is completely superseded by `postgresql_dev_env` in all usage sites